### PR TITLE
rebalance report

### PR DESCRIPTION
**Pull request by gallizoltan**
- `rebalancereport`: shows information about rebalance
- some README update

### CODE REVIEW

File: rebalance/rebalance.py [hunk 1]
```diff
@@ -666,6 +666,46 @@ def rebalancestop(plugin: Plugin):
...
 
+@plugin.method("rebalancereport")
+def rebalancereport(plugin: Plugin):
+    """Show information about rebalance
+    """
+    res = {}
+    res["rebalanceall_is_running"] = plugin.mutex.locked()
+    res["getroute_method"] = plugin.getroute.__name__
+    res["maxhops_threshold"] = plugin.maxhops
+    res["msatfactor_threshold"] = plugin.msatfactor
```
Comment (m-schmoock) [inline rebalance/rebalance.py:677]: this is not a threshold value ;)  We can name it just `msatfactor` or `capacity_factor`. In the latter case we should also rename it in the plugin remaining code. I think its a better speaking name.

**Comment by m-schmoock** [comment]
Nice :+1:

File: rebalance/rebalance.py (hunk 1 above)
Comment (gallizoltan) [reply rebalance/rebalance.py]: I started to write `threshold` everywhere, because I thought the `erringnodes` expression is misleading here. :)

Comment (m-schmoock) [reply rebalance/rebalance.py]: Yes, the names aren't optimal

File: rebalance/rebalance.py [hunk 2]
```diff
@@ -666,6 +666,46 @@ def rebalancestop(plugin: Plugin):
...
+        try:
+            pay = plugin.rpc.listpays(r["bolt11"])["pays"][0]
+            total_amount += pay["amount_msat"]
+            total_fee += pay["amount_sent_msat"] - pay["amount_msat"]
+        except Exception:
+            res["total_successful_rebalances"] -= 1
+    res["total_rebalanced_amount"] = total_amount
+    res["total_rebalance_fee"] = total_fee
+    if total_amount > Millisatoshi(0):
+        res["average_rebalance_fee_ppm"] = round(total_fee/total_amount*10**6, 2)
```
Comment (m-schmoock) [inline rebalance/rebalance.py:703]: That is nice :D

**Comment by m-schmoock** [comment]
tACK

Good to have this, maybe we can improve on this and add more metrics

**Pull request merged**